app/models/events/comment_replied_to.rb

```diff
@@ -1,16 +1,9 @@
 class Events::CommentRepliedTo < Event
   def self.publish!(comment)
-    return unless comment.parent.present?
-    event = create!(kind: 'comment_replied_to',
-                    eventable: comment,
-                    created_at: comment.created_at)
-
-    if comment.parent.author != comment.author
-      ThreadMailer.delay.comment_replied_to(comment.parent.author, event) if comment.parent.author.email_when_mentioned?
-      event.notify! comment.parent.author
-    end
-
-    event
+    return unless comment.parent.present? && comment.parent.author != comment.author
```

Comment:
This is a slight change in behaviour, but it's better I think. This means we don't even create an event for comment_replied_to when the replier is the same as the reply author.

Before we were creating the event, but not notifying anyone about it.